### DISCUSSION TITLE
Livewire lifecycle hooks not firing inside nested component

### DISCUSSION THREAD

**Discussion by Quadrubo** [livewire/livewire#7000] [1. Help]
Hey, I have 2 livewire components. One is called `Map` and has the property `$filters`. The other is called `MapWidget` and get's the property from the `Map` component. Now I want to call code when the `$filters` property of the child component `MapWidget` updates. However the hooks don't seem to fire.
Map:
```php
<?php
class Map extends Livewire
{
    public ?array $filters = [];

    ...
}
```
```html
<form wire:submit='create'>
    ...
    
    <button type="submit">
        Submit
    </button>
</form>

<div>
    @livewire(\App\Livewire\MapWidget::class, ['filters' => $filters])
</div>
```

MapWidget
```php
class MapWidget extends Livewire
{
    public ?array $filters = [];

    public function updated($property)
    {
        // $property: The name of the current property that was updated
 
        dd($property);
    }
}
```

When putting the `updated` function into the parent component it works as expected. I can also see that the data changes when  I echo it in the view. The only thing not working are these hooks.

**Comment by joshhanley** [comment]
@Quadrubo why are you making them separate components? I would just put that all in one component. The updated hook will only run in the component where the property gets changed (and only when changed from the front end, like using wire:model). I can't see your view for the MapWidget so I'm not exactly sure how you're interacting with the property.
